src/mappings/chainMigrations/legacyStats.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * Assets issued before v5.0 had an implicitly made Count statistics
+ *
+ * This is a list for testnet and mainnet obtained with:
+ *
+ * ```sh
+ * curl 'http://localhost:3007/' -H 'Accept-Encoding: gzip, deflate, br' -H 'Content-Type: application/json' -H 'Accept: application/json' -H 'Connection: keep-alive' -H 'DNT: 1' -H 'Origin: http://localhost:3007' --data-binary '{"query":"query issued {\n  events(filter: {blockId: {lessThan: \"4397818\"},eventId: {equalTo: Issued}}) {\n    nodes{\n      id\n      eventArg0\n      eventArg1\n      eventArg2\n      eventArg3\n    }\n  }\n}\n"}' --compressed | jq '.data.events.nodes | map(.eventArg1) | unique' | tee /tmp/asset_list.txt
```

Comment:
This query will not give correct data as the blockId column is string and querying on that didn't give correct data. For any block starting with number greater than 4 like 500 or 60001 or 99999 etc, those blocks will get skipped by this.

Reply:
Thanks. I was wondering what was wrong with the query.

Any ideas to stop other users from making them same mistake?

Reply:
I'm not sure. There should a way around the graphql to first filter out blocks and then fetch in the events but I'm not sure. I'm not that good at clubbing graphql queries